### PR TITLE
Throttler: Minor changes to make throttler work with native safari

### PR DESCRIPTION
**Pull request by mrocajr**
## Description
Please describe the change as necessary.
Minor changes to make throttler work with native safari

## Specific Changes proposed
ts file headers set to application/vnd.apple.mpegurl

### CODE REVIEW

File: utils/switcher/throttle-server.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ const writeSlowly = (req, res, data) => {
 // maps file extention to MIME type
 // we only care about these two types
 const map = {
-  '.ts': 'application/octet-stream',
+  '.ts': 'application/vnd.apple.mpegurl',
```
Comment (gesinger) [inline utils/switcher/throttle-server.js]: `vnd.apple.mpegurl` denotes an m3u8 playlist

Comment (mrocajr) [inline utils/switcher/throttle-server.js]: Perhaps video/MP2T would be more appropriate?

Comment (gesinger) [inline utils/switcher/throttle-server.js]: That would be good

File: utils/switcher/throttle-server.js [hunk 2]
```diff
@@ -95,8 +95,8 @@ const writeSlowly = (req, res, data) => {
 // maps file extention to MIME type
 // we only care about these two types
 const map = {
-  '.ts': 'application/octet-stream',
-  '.m3u8': 'application/x-mpegurl'
+  '.ts': 'application/vnd.apple.mpegurl',
```
Comment (gesinger) [inline utils/switcher/throttle-server.js]: I think these two are reversed. And did these have to be changed to get Safari to work?

Comment (mrocajr) [reply utils/switcher/throttle-server.js]: Yep, just noticed that 👍

**Comment by gesinger** [comment]
Tested, will merge once build passes.

**Pull request merged**